### PR TITLE
chore: make website copy up to standards

### LINKED ISSUE #25: chore: change copy to new standards
We can have Jason look at our website and make a new copy for it!

* [x] :wave: I may be able to implement this feature request

### PR DESCRIPTION
**Pull request by BryanPan342**
update website copy

Fixes #25

### CODE REVIEW

File: src/assets/sectionInfo.json [hunk 1]
```diff
@@ -2,46 +2,46 @@
   "about": [
     {
       "image": "who-are-we.svg",
-      "title": "Who are we?",
-      "body": "The Creative Labs community is managed by a small team of developers, designers, and project managers.",
-      "linkText": "More on our team + history ➔",
-      "linkPath": "",
+      "title": "Welcome to Our Lab",
+      "body": "Just some UCLA kids with skills in dev, design, and project management trying to make something cool, together. We were founded on the dream that with the right group of people, any idea can come to life.",
+      "linkText": "Join our Community ->",
```
Comment (arjunr00) [inline src/assets/sectionInfo.json:7]:
```suggestion
      "linkText": "Join our Community ➔",
```

File: src/assets/sectionInfo.json [hunk 2]
```diff
@@ -2,46 +2,46 @@
...
     {
       "image": "what-we-do.svg",
-      "title": "What do we do?",
-      "body": "Creative Labs gives students the platform that allows them to make their ideas come to life. We were founded on the dream to just make cool things.",
-      "linkText": "Join the community ➔",
-      "linkPath": "",
-    },
+      "title": "Creativity for All",
+      "body": "We bring students from all disciplines and backgrounds together to work on technical & creative projects.",
+      "linkText": "Our History & Team ->",
```
Comment (arjunr00) [inline src/assets/sectionInfo.json:14]:
```suggestion
      "linkText": "Our History & Team ➔",
```

File: src/assets/sectionInfo.json [hunk 3]
```diff
@@ -2,46 +2,46 @@
...
-    {
-      "image": "spring.svg",
-      "title": "Spring 2020",
-      "body": "Completed projects during our very first virtual quarter.",
-    },
     {
       "image": "winter.svg",
       "title": "Winter 2020",
-      "body": "ultra short blurb",
+      "body": ""
```
Comment (arjunr00) [inline src/assets/sectionInfo.json:22]: Just double checking that we won't have a blurb here?

**Review by arjunr00 — COMMENTED** [review]
Just a couple of nits but otherwise LGTM on my phone, will check it out on my computer in the morning and slap Approve

File: src/assets/sectionInfo.json (hunk 3 above)
Comment (BryanPan342) [reply src/assets/sectionInfo.json:22]: i think its better to have nothing as opposed to ultra-short blurb tbh

Comment (arjunr00) [reply src/assets/sectionInfo.json:22]: fair enough

**Review by arjunr00 — APPROVED** [review]

**Review by arjunr00 — APPROVED** [review]

**Pull request merged**